src/__tests__/real-world-example/__fixtures__/README.md

```diff
@@ -0,0 +1,30 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
Is the default readme left here intentionally?

Reply:
Yes, I have generated app with `npx create-next-app`; idea is that I'm able to go into that directory and run `yarn next dev` and compare `html` from server with what we render.